Makefile

```
@@ -373,15 +373,15 @@ MANPAGES_MD ?= $(wildcard docs/source/markdown/*.md pkg/*/docs/*.md)
 MANPAGES ?= $(MANPAGES_MD:%.md=%)
 MANPAGES_DEST ?= $(subst markdown,man, $(subst source,build,$(MANPAGES)))
 
-$(MANPAGES): %: %.md .gopathok
+$(MANPAGES): %: %.md .install.md2man docdir
```

Should the `.gopathok` really be removed?  Otherwise LGTM.

```suggestion
$(MANPAGES): %: %.md .gopathok .install.md2man docdir
```

---

Yes, I think so, since .install.md2man target has already depended on it, see:
Makefile:
```
.install.md2man: .gopathok
```